File: BackEnd/Kotlin/projects/jtok-starter/src/main/java/Lec05Main.java
public class Lec05Main {
  public static void main(String[] args) {

  }

  private void validateScoreIsNotNegative(int score) {
    if (score < 0) {
      throw new IllegalArgumentException(String.format("%d는 0보다 작을 수 없습니다.", score));
    }
  }

  private String getPassOrFail(int score) {
    if (score >= 50) {
      return "P";
    } else {
      return "F";
    }
  }
}

File: BackEnd/Kotlin/projects/jtok-starter/src/main/kotlin/Lec05Main.kt
fun validateScoreIsNotNegative(score: Int) {
  if (score < 0) {
    throw IllegalArgumentException("${score}는 0보다 작을 수 없습니다.")
  }
}

fun getPassOrFail(score: Int): String {
  return if (score >= 50) {
    "P"
  } else {
    "F"
  }
}